.github/workflows/docker_build.yml

```
@@ -0,0 +1,105 @@
+name: "Appserver: Build & Push Docker Image"
+run-name: Build & Push appserver/${{ github.base_ref || github.ref_name }}
+on:
+  push:
+    branches: [main]
+defaults:
+  run:
+    shell: bash
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: harbor.delivery.iqgeo.cloud/${{ vars.harbor_project_name }}
+      platform_version: 7.2
+    steps:
+      -
+        name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      -
+        name: Authenticate Docker to Harbor
+        uses: docker/login-action@v3
+        with:
+          registry: harbor.delivery.iqgeo.cloud
+          username: ${{ secrets.container_registry_username }}
+          password: ${{ secrets.container_registry_password }}
+
+      -
+        name: "Build: Platform"
+        id: platform
+        uses: docker/build-push-action@v6
+        with:
+          push: false
+          pull: false
+          context: .
+          file: deployment/dockerfile.build
+          build-args: |
+            CONTAINER_REGISTRY=${{ env.REGISTRY }}
+          platforms: linux/amd64,linux/arm64
+          tags: |
+            ${{ env.REGISTRY }}/platform-build:${{ env.platform_version }}
+            iqgeo-myproj-build
+          cache-to: type=gha,mode=max
+          cache-from: type=gha
+
+      -
+        name: Extract Appserver metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.REGISTRY }}/platform-appserver
+          tags: |
+            type=raw,value={{branch}}-{{sha}}
+            type=sha,prefix={{branch}}-,enable={{!is_default_branch}}
+            type=raw,value=${{ env.platform_version }},enable={{is_default_branch}}
+
+      -
+        name: "Build: Appserver"
+        uses: docker/build-push-action@v6
+        with:
+          push: true
+          pull: true
+          context: deployment
+          file: deployment/dockerfile.appserver
+          # secret-files: |
+          #   oidc=${{ secrets.oidc_config }}
```

```suggestion
          # secret-files: |
          #   oidc=${{ secrets.oidc_config }}
```

This, along with the included changes recommended to the [`dockerfile.appserver`](deployment/dockerfile.appserver) would facilitate a secure method for providing the OIDC configuration to the build. (An exclusively GitHub Actions flavored method, but it would serve as a guide regardless.)